src/main.rs

```diff
@@ -261,6 +262,12 @@ async fn run_server(addr: SocketAddr) -> anyhow::Result<()> {
         octocrab: oc,
     });
 
+    // Run all jobs that don't have a schedule (one-off jobs)
+    if !is_scheduled_jobs_disabled() {
```

Comment:
Hmm, what is this function

Reply:
Oh, its preexisting. Okay.

Reply:
I'm not going to block this PR on it, but this should be able to be in the same if block as below. Please fix next PR